x-pack/plugins/security_solution/public/common/containers/first_last_seen/index.tsx

```diff
@@ -10,78 +10,81 @@ import { useCallback, useEffect, useRef, useState } from 'react';
 import { Subscription } from 'rxjs';
 
```

Comment:
This directory filenames are a little bit inconsistent. The hook implementation is in the `index.tsx` (we decided not to do so) but the hook test is called `first_last_seen.test.tsx`. Also, there's no need to have the `.tsx` extension, there's no jsx in them. My suggestion:
- first_last_seen.test.ts
- first_last_seen.ts (hook implementation)
- index.ts (export only)

Reply:
fixed :D